src/data/local/github/tracks-node/track-3/steps/step-1.md

```diff
@@ -5,12 +5,12 @@ In this track, you will implement a login mutation fully integrated with the dat
 
 Let's start with baby steps, by creating the mutation prototype, with no integration with the dabatase (yet).
 
-Let's call the mutation `Login`. It should receive an e-mail and a password as parameters and return the following structure on body:
+Let's call the mutation `login`. It should receive an e-mail and a password as parameters and return the following structure on body:
```

Comment:
```suggestion
Let's call the mutation `login`. It should receive an e-mail and a password as parameters and, in case of success, 
 should return the following structure:
```